params/params.go

```diff
@@ -138,6 +138,7 @@ type Tag struct {
 
 type Pool struct {
 	ID                     string        `json:"id"`
+	RunnerPrefix		   string        `json:"runner_prefix"`
```

Comment:
Please run ```gofmt -w -s``` on changed files. I find it useful to set up the IDE/Code editor to do it automatically. I think the vs code golang plugin does this out of the box.

Reply:
sorry

Reply:
No worries at all :smile: